test/fixtures.yml

```diff
@@ -371,6 +371,11 @@ jetpay:
 komoju:
   login: sk_f1dd75ce3d5cad477eac0c827c1cac8eaa51ede3
 
+latitude19:
```

Comment:
@rwdaigle
 
Not 100% sure about this part. Right now the creds (`account_number, configuration_id, secret`) work only on their sandbox. From the few conversations with Mike, it seemed to me like the same `account_number` and `configuration_id` may get carried over to production. But, probably not the `secret`. Other aspect is that, these values have been setup particularly for Fonteva. So, bit unsure if this can be commiitted to the open repo.